protocol/src/test/java/org/opensearch/sql/protocol/response/format/VisualizationResponseFormatterTest.java

```diff
@@ -187,4 +187,10 @@ private static void assertJsonEquals(String expected, String actual) {
         JsonParser.parseString(expected),
         JsonParser.parseString(actual));
   }
+
+  @Test
+  void testContentType() {
+    var formatter = new CommandResponseFormatter();
+    assertEquals(formatter.contentType(), "application/json; charset=UTF-8");
```

Comment:
if you use CONTENT_TYPE here, then your tests will pass even if the header text changes.